draft-ietf-ohai-ohttp.md

```diff
@@ -974,12 +976,33 @@ examine the content of a request (other than to observe the choice of key
 identifier, KDF, and AEAD), so it is also assumed that it cannot generate an
 encapsulated response.
 
-A proxy MUST NOT add information about the client identity when forwarding
-requests. This includes the Via field, the Forwarded field
-{{?FORWARDED=RFC7239}}, and any similar information.  A client does not depend
-on the proxy using an authenticated and encrypted connection to the oblivious
-request resource, only that information about the client not be attached to
-forwarded requests.
+A proxy SHOULD NOT add information about the client identity when forwarding
+requests. For example, including the Via field, the Forwarded field
+{{?FORWARDED=RFC7239}}, or similar information may reveal information about the
+client that the proxy is intended to hide from the target.  In general, a client
+does not depend on the proxy using an authenticated and encrypted connection to
+the oblivious request resource, only that information about the client not be
+attached to forwarded requests. However, some applications or use cases can
+benefit additional client metadata attached to forwarded request. Some example
+HTTP headers that the proxy might include are as follows:
+
+- Flag indicating whether or not the client is deemed malicious. If a client
+  request orignates from an abusive or otherwise harmful IP address with a bad
+  reputation, the proxy might attached a flag indicating this to the target. The
+  target could use this flag to shadowban clients or other alter whether how
+  the client encapuslated request is processed.
+- Convey coarse geographic information about the client. Some applications, such
```

Comment:
...and there's that thin end.

I know why you want this, but I think that it is a very bad idea.

I can see how some clients might agree to have this information exposed, but geolocation is a privacy minefield.

I would rather set the terms here in general rather than seek to describe specific (bad) exclusions our general "no client information" policy.  Essentially, establish that clients can, with prior agreement with a proxy, describe the sort of information that a proxy might reveal about them.